lib/net.js

```diff
@@ -760,7 +760,7 @@ function afterWrite(status, handle, req, err) {
   }
 
   if (status < 0) {
-    var ex = exceptionWithHostPort(status, 'write', req.address, req.port);
+    var ex = errnoException(status, 'write', req.error);
```

Comment:
Why is this change necessary?  Also, I think this a) makes it a semver-major, and b) users will be less than pleased with the loss of the address/port information.